app/src/main/res/values/strings.xml

```diff
@@ -82,18 +82,40 @@
         <string name="pending_intent">Pending intent</string>
         <string name="btn_txt_navigation_controller">Navigation Controller</string>
         <string name="title_activity_pending">PendingActivity</string>
-        <!-- Strings used for fragments for navigation -->
         <string name="first_fragment_label">First Fragment</string>
         <string name="second_fragment_label">Second Fragment</string>
         <string name="next">Next</string>
         <string name="previous">Previous</string>
-
         <string name="hello_first_fragment">Hello first fragment</string>
         <string name="hello_second_fragment">Hello second fragment. Arg: %1$s</string>
         <string name="btn_txt_start_pending_intent">start Pending Intent</string>
         <string name="tabone_string">Tab 1</string>
         <string name="tabtwo_string">Tab Two</string>
         <string name="mainactivity_tag">MainActivityLog</string>
         <string name="fragment_count">FragmentCountLog</string>
+        <string name="mainactivity_log_tag">MainActivityLog</string>
+        <string name="activity_log_tag">ActivitytwoLog</string>
+        <string name="google_url">https://www.google.com/</string>
+        <string name="txt_data_name">Txt data</string>
+        <string name="name_key_intent">name</string>
+    <string name="lbl_Fragment_Activity">Fragment Activity</string>
+        <string name="lbl_Intent_Activity">Intent Activity</string>
+        <string name="lbl_ExpandableListview_Activity">ExpandableListview Activity</string>
+        <string name="lbl_ExpandableRecyclerview_Activity">ExpandableRecyclerview Activity</string>
+        <string name="lbl_ViewpagertwoRecyclerview_Activity">ViewpagertwoRecyclerview Activity</string>
+        <string name="lbl_ViewPagertwo_Activity">ViewPagertwo Activity</string>
+        <string name="lbl_ViewPagerone_Activity">ViewPagerone Activity</string>
+        <string name="lbl_GridLayoutRecycler_ViewActivity">GridLayoutRecycler ViewActivity</string>
+        <string name="lbl_ConstraintLayoutGuideline_Activity">ConstraintLayoutGuideline Activity</string>
+        <string name="lbl_RecyclerView_Activity">RecyclerView Activity</string>
+        <string name="lbl_Listview_Activity">Listview Activity</string>
+        <string name="lbl_IncludeandMerge_Activity">IncludeandMerge Activity</string>
+        <string name="lbl_GridLayout_Activity">GridLayout Activity</string>
+        <string name="lbl_FrameLayout_Activity">FrameLayout Activity</string>
+        <string name="lbl_Layout_Activity">Layout Activity</string>
+        <string name="lbl_ConstraintLayout_Activity">ConstraintLayout Activity</string>
```

Comment:
underscore pattern don't use camel case letter. Change in whole file. ex:  `lbl_constraint_layout_activity`